actors/power/src/lib.rs

```diff
@@ -289,7 +292,7 @@ impl Actor {
 
     fn submit_porep_for_bulk_verify(
         rt: &mut impl Runtime,
-        seal_info: SealVerifyInfo,
```

Comment:
Reiterating: I think it makes sense for methods to define param names that make sense in context